lib/games_teams_collection.rb

```diff
@@ -59,21 +59,23 @@ def percentage_ties
   end
 
   def number_of_wins
```

Comment:
Could we rename this so it reflects what it's doing now?

Reply:
hhhhmmm it doesnt sound like what it should?